### PR TITLE
Qa/updating access control 2 1

### PR DESCRIPTION
**Pull request by msdolbey**
## Jira Ticket
[D2IQ-78900](https://jira.d2iq.com/browse/D2IQ-78900)

## Description of changes being made
Updating this page to be in line with what is in the app for 2.1. This is a carryover ticket from porting stuff from 1.x to 2.x. This was mostly there, but some updated images and some fixes.

`backport needed` to 2.2. So ... forward port? Anyway - once this is in, we can update 2.2

## Checklist

- [x] Test all commands and procedures, if applicable.
- [x] Create new PRs against `develop`, or `main` for hotfixes to production.
- [x] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by mwrightd2 — APPROVED** [review]

File: pages/dkp/kommander/2.1/operations/access-control/index.md [hunk 1]
```diff
@@ -22,15 +22,15 @@ Use these two groups of resources to manage access control within 3 levels of sc
 | Workspace | Create namespaced Roles on the management cluster in the workspace namespace. | Federates ClusterRoles on all target clusters in the workspace.                            |
 | Project   | Create namespaced Roles on the management cluster in the project namespace.   | Federates namespaced Roles on all target clusters in the project in the project namespace. |
 
-The policies for each level and type create RoleBindings or ClusterRoleBindings on the clusters that apply to each category.
+The role bindings for each level and type create `RoleBindings` or `ClusterRoleBindings` on the clusters that apply to each category.
```
Comment (cneth) [inline pages/dkp/kommander/2.1/operations/access-control/index.md:25]: Is "role bindings" really the right term change here.   The way I read this text, the user is creating "Kommander Roles" and "Cluster Roles" at the different 'levels', and then Kommander creates the RoleBindings/ClusterRoleBindings?     So the user is not creating 'role bindings'???

**Review by cneth — APPROVED** [review]

File: pages/dkp/kommander/2.1/operations/access-control/index.md (hunk 1 above)
Comment (msdolbey) [reply pages/dkp/kommander/2.1/operations/access-control/index.md:25]: Well, they are role bindings as we define them.

We changed the terms/phrasing in the UI specifically from policies to Role Bindings. So, whenever it was referencing Policies, we should have it changed to role bindings.

File: pages/dkp/kommander/2.1/operations/access-control/index.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ menuWeight: 10
 excerpt: Centrally manage access across clusters
 ---
 
-You can centrally define role-based authorization within Kommander to control resource access on the management cluster and a set or all of the target clusters. These resources are similar to Kubernetes RBAC but with crucial differences, and they make it possible to define the roles and policies once and have them federated to clusters within a given scope.
+You can centrally define role-based authorization within Kommander to control resource access on the management cluster and a set or all of the target clusters. These resources are similar to Kubernetes RBAC but with crucial differences, and they make it possible to define the roles and role bindings once, and have them federated to clusters within a given scope.
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.1/operations/access-control/index.md:10]:
```suggestion
You can centrally define role-based authorization within Kommander to control resource access on the management cluster and a set, or all, of the target clusters. These resources are similar to Kubernetes RBAC but with crucial differences, and they make it possible to define the roles and role bindings once, and have them federated to clusters within a given scope.
```

File: pages/dkp/kommander/2.1/operations/access-control/index.md [hunk 3]
```diff
@@ -67,20 +67,23 @@ subjects:
 
 ## Types of Access Control Objects
 
-Kubernetes role-based access control can be controlled with three different object categories: Groups, Roles and Policies, as explained in more detail below.
+Kubernetes role-based access control can be controlled with three different object categories: Groups, Roles, and Role Bindings, as explained in more detail below.
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.1/operations/access-control/index.md:70]:
```suggestion
You can control Kubernetes role-based access control with three different object categories: Groups, Roles, and Role Bindings, as explained in more detail below.
```

File: pages/dkp/kommander/2.1/operations/access-control/index.md [hunk 4]
```diff
@@ -67,20 +67,23 @@ subjects:
...
 
-You can map group and user claims made by your configured identity providers to Kommander groups by clicking the _Groups_ tab under Administration / Identity providers on the global level.
+You can map group and user claims made by your configured identity providers to Kommander groups by clicking the _Groups_ tab under Administration / Identity providers on the global workspace level.
 
 ![Identity Provider Groups](/dkp/kommander/2.1/img/access-control-idp-groups.png)
 
 ### Roles
 
-ClusterRoles are named collections of rules defining which verbs can be applied to which resources.
+`ClusterRoles` are named collections of rules defining which verbs can be applied to which resources.
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.1/operations/access-control/index.md:80]: This seems out of place way down here. Seems like this explanation should be offered earlier in this topic.

File: pages/dkp/kommander/2.1/operations/access-control/index.md [hunk 5]
```diff
@@ -67,20 +67,23 @@ subjects:
...
 
-- Kommander Roles apply specifically to resources on the management cluster.
-- Cluster Roles apply to target clusters within their scope: at the global level, this is all target clusters in all workspaces, at the workspace level this is all target clusters in the workspace, at the project level this is all target clusters that have been added to the project.
+-   Kommander Roles apply specifically to resources on the management cluster.
+-   Cluster Roles apply to target clusters within their scope:
+    - at the global level, this is all target clusters in all workspaces,
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.1/operations/access-control/index.md:84]:
```suggestion
    - Global - all target clusters in all workspaces
```

File: pages/dkp/kommander/2.1/operations/access-control/index.md [hunk 6]
```diff
@@ -67,20 +67,23 @@ subjects:
...
+    - at the workspace level, this is all target clusters in the workspace,
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.1/operations/access-control/index.md:85]:
```suggestion
    - Workspace - all target clusters in the workspace
```

File: pages/dkp/kommander/2.1/operations/access-control/index.md [hunk 7]
```diff
@@ -67,20 +67,23 @@ subjects:
...
 
 ### Roles
 
-ClusterRoles are named collections of rules defining which verbs can be applied to which resources.
+`ClusterRoles` are named collections of rules defining which verbs can be applied to which resources.
 
-- Kommander Roles apply specifically to resources on the management cluster.
-- Cluster Roles apply to target clusters within their scope: at the global level, this is all target clusters in all workspaces, at the workspace level this is all target clusters in the workspace, at the project level this is all target clusters that have been added to the project.
+-   Kommander Roles apply specifically to resources on the management cluster.
+-   Cluster Roles apply to target clusters within their scope:
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.1/operations/access-control/index.md:83]:
```suggestion
-   Cluster Roles apply to target clusters within their scope at these levels:
```

File: pages/dkp/kommander/2.1/operations/access-control/index.md [hunk 8]
```diff
@@ -136,20 +139,19 @@ subjects:
     name: oidc:engineering
 ```
 
-### Policies
+### Role Bindings
 
-Kommander policies, cluster policies and project policies bind a Kommander group to any number of roles.
-All groups that have been defined in the groups tab will be present at the global, workspace, or project level and are ready for you to assign roles to them.
+Kommander role bindings, cluster role bindings, and project role bindings bind a Kommander group to any number of roles.
+All groups that have been defined in the groups tab will be present at the global, workspace, or project level, and are ready for you to assign roles to them.
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.1/operations/access-control/index.md:145]:
```suggestion
All groups defined in the `groups` tab will be present at the global, workspace, or project level, and are ready for you to assign roles to them.
```

**Review by JohnB64-JBV — APPROVED** [review]

File: pages/dkp/kommander/2.1/operations/access-control/index.md (hunk 4 above)
Comment (msdolbey) [reply pages/dkp/kommander/2.1/operations/access-control/index.md:80]: There is a bit of discussion a bit before here - it's at least in this table that's above, so I think there is some start to it.

This topic is ... essentially a few topics inside the overall Access Control area.

File: pages/dkp/kommander/2.1/operations/access-control/index.md (hunk 1 above)
Comment (cneth) [reply pages/dkp/kommander/2.1/operations/access-control/index.md:25]: Sorry, should have made it clearer.  I understand you were just updating 'policies->role bindings'.   I am wondering if the word 'policies' here was being used in a more generic sense before or perhaps it was misused or just wrong...

When I read this whole section in total, I don't see any reference in the words above it to either 'policies' _or_ 'role bindings'.   The first paragraph in this section says "Kommander has two conceptual groups of resources" and then mentions Kommander Roles and Cluster Roles, and then there is a table that describes how Kommander Roles are mapped to ClusterRoles based on what context they are in.    

I guess I'm saying that this sentence makes no sense to me with either subject!  What am I missing?

File: pages/dkp/kommander/2.1/operations/access-control/index.md (hunk 5 above)
Comment (msdolbey) [reply pages/dkp/kommander/2.1/operations/access-control/index.md]: made this change with a later commit

File: pages/dkp/kommander/2.1/operations/access-control/index.md (hunk 7 above)
Comment (msdolbey) [reply pages/dkp/kommander/2.1/operations/access-control/index.md]: made this change with a later commit

File: pages/dkp/kommander/2.1/operations/access-control/index.md (hunk 8 above)
Comment (msdolbey) [reply pages/dkp/kommander/2.1/operations/access-control/index.md]: made this change with a later commit

File: pages/dkp/kommander/2.1/operations/access-control/index.md (hunk 6 above)
Comment (msdolbey) [reply pages/dkp/kommander/2.1/operations/access-control/index.md]: made this change with a later commit

File: pages/dkp/kommander/2.1/operations/access-control/index.md (hunk 1 above)
Comment (msdolbey) [reply pages/dkp/kommander/2.1/operations/access-control/index.md:25]: ahh - the big reveal - I see what you're saying. I think John made a similar message below. I'll try to rework. Thanks for clarifying.

Comment (msdolbey) [reply pages/dkp/kommander/2.1/operations/access-control/index.md:25]: I think that role bindings is more of a generic Kubernetes term - but this is an intro topic anyway, so it should be explained in some way.

**Review by cneth — APPROVED** [review]

**Pull request merged**